lib/vector-core/Cargo.toml

```diff
@@ -95,7 +95,7 @@ rand = "0.8.5"
 rand_distr = "0.4.3"
 tracing-subscriber = { version = "0.3.17", default-features = false, features = ["env-filter", "fmt", "ansi", "registry"] }
 vector-common = { path = "../vector-common", default-features = false, features = ["test"] }
-vrl.workspace = true
+vrl = { version = "0.6.0", features = ["cli", "test", "test_framework", "arbitrary"] }
```

Comment:
That's pretty unfortunate that we can't just add a feature to the `workspace` version :/